### PR TITLE
Seo improvement to devel

### PR DESCRIPTION
**Pull request by ondrasalek**
SEO...

Metatags...
Bigger Images edit like png/jpg to webp.
some accesibility edits like colors contrast and some spaces around links and buttons

### CODE REVIEW

**Review by SedlarDavid — APPROVED** [review]
Very najs job, thank you for that!

File: src/components/layout/header/navbar/Navbar.tsx [hunk 1]
```diff
@@ -6,13 +6,14 @@ import { useLocation } from 'react-router-dom';
...
 const Navbar = ({ isOpen }: INavbarProps) => {
     const location = useLocation();
-
     return (
         <Wrapper isOpen={isOpen}>
             {LINKS.map((link, index) => (
-                <NavLink href={link.linkTo} key={index} active={location.pathname === link.linkTo}>
-                    {link.text}
-                </NavLink>
+                <li key={index} >
```
Comment (tomesfilip) [inline src/components/layout/header/navbar/Navbar.tsx:12]: maybe Navlink could cosist of the li tag so there is no need to add it everywhere like this

File: src/components/layout/cookies/Cookies.tsx [hunk 2]
```diff
@@ -14,15 +14,21 @@ const Cookies = () => {
...
-                <BaseLink underline href="https://www.cookie-lista.cz/co-je-cookies.html" target="_blank" rel="noreferrer">
-                    Více o cookies
-                </BaseLink>
-                <BaseButton style onClick={() => handleCookies("hide")}>
-                    Odmítnout
-                </BaseButton>
-                <BaseButton style onClick={() => handleCookies("accept")}>
-                    Souhlasím s cookies
-                </BaseButton>
+                <li>
```
Comment (tomesfilip) [inline src/components/layout/cookies/Cookies.tsx:17]: Baselink could consist of li tag to avoid using it explicitly everywhere

**Review by tomesfilip — APPROVED** [review]
few possible changes, nothing breaking, gj

**Pull request merged**